packages/angular/cli/src/utilities/completion.ts

```diff
@@ -72,12 +73,29 @@ Ok, you won't be prompted again. Should you change your mind, the following comm
   // Notify the user autocompletion was set up successfully.
   logger.info(
     `
-Appended \`source <(ng completion script)\` to \`${rcFile}\`. Restart your terminal or run the following to autocomplete \`ng\` commands:
+${tags.oneLine`
+  Appended \`source <(ng completion script)\` to \`${rcFile}\`. Restart your terminal or run the
+  following to autocomplete \`ng\` commands:
+`}
 
     ${colors.yellow(`source <(ng completion script)`)}
     `.trim(),
   );
 
+  if ((await hasGlobalCliInstall()) === false) {
+    logger.warn(
+      `
+${tags.oneLine`
```

Comment:
I think this would cause too much clutter and make it hard to read if it was in a single line.

Maybe better have it multiline?

Reply:
Chatted about this offline a bit. The goal here was to avoid introducing `\n` into the string for long paragraphs of text since it forces line wrapping in a terminal we don't know the full width of. I was hoping `tags.oneLine` would make that a little easier, but trying again with concatenated strings seems a bit more readable, so I went with that approach.